redis-om-spring/src/main/java/com/redis/om/spring/convert/BooleanToBytesConverter.java

```diff
@@ -0,0 +1,23 @@
+package com.redis.om.spring.convert;
+
+import java.nio.charset.StandardCharsets;
+
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.data.convert.WritingConverter;
+import org.springframework.stereotype.Component;
+
+@Component
+@WritingConverter
+public class BooleanToBytesConverter implements Converter<Boolean, byte[]> {
+  byte[] fromString(String source) {
+    return source.getBytes(StandardCharsets.UTF_8);
```

Comment:
How can you be sure they're UTF_8?

Reply:
Following the Spring Data Redis lead, we serialize to UTF8 and deserialize from UTF8. We control the serialization charset used in and out of the ORM layer.